content/info/hardware.md

```diff
@@ -0,0 +1,28 @@
++++
+title = "Hardware development"
+date = 2024-01-13
++++
+
+## Contract hardware development
+
+We offer contract hardware development of high complexity. Expertise includes but is not limited to:
```

Comment:
and then

> We offer contract hardware development of high complexity.

We offer contract hardware development and consulting services for complex projects.

Reply:
why not use suggestions?

```suggestion
We offer contract hardware development and consulting services for complex projects. Expertise includes but is not limited to:
```

Reply:
noted, never tried it before, will try in the future